src/utils/voice/helpers/ai.ts

```
@@ -1,18 +1,18 @@
+import { regularPrompt } from '@/lib/ai/prompts';
+import { myProvider } from '@/lib/ai/providers';
+import { generateText } from 'ai';
+
+// TODO: Add Memories, and other tools available in the AI provider
+// TODO: Add History from the VC Chat Channel
+// TODO: Add a better voice prompt, and also switch to 11labs v3 as the voice is much better
+export async function getAIResponse(prompt: string): Promise<string> {
+  const { text } = await generateText({
+    system:
+      regularPrompt +
+      '\n\nYou are talking to a person through a call, do not use markdown formatting, or emojis.',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)